recipe/meta.yaml

```diff
@@ -1,41 +1,42 @@
-{% set version = "1.6.3" %}
+{% set name = "g2clib" %}
+{% set version = "1.7.0" %}
 
 package:
-  name: g2clib
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: http://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/g2clib-{{ version }}.tar
-  sha256: 83e4f2061b3d4a8bd431ba860dda8c5cf103b25f42d265b7b724b9acafad177c
+  url: https://github.com/NOAA-EMC/NCEPLIBS-g2c/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 73afba9da382fed73ed8692d77fa037bb313280879cd4012a5e5697dccf55175
 
 build:
-  number: 1
-  skip: True  # [win]
```

Comment:
@timcera, the reason you are seeing a windows build now is because you removed this line that skips Windows.  Could you add it back?